src/cromshell/status/command.py

```diff
@@ -0,0 +1,123 @@
+import logging
+import click
+
+import requests
+import tempfile
+import json
+import re
+import fileinput
+import jq
+
+from cromshell.utilities import HTTPUtils
+from cromshell.utilities import IOUtils
+
+LOGGER = logging.getLogger(__name__)
+
+
+@click.command(name="status")
+@click.option(
+    "-q",
+    "--quiet",
+    is_flag=True,
+    help="Suppress Turtle Logo",
+)
+@click.argument('workflow_id')
+@click.pass_obj
+def main(cromwell_config, workflow_id, quiet):
+    """Check the status of a Cromwell job UUID"""
+    click.echo("status")
+    LOGGER.info("status")
+
+    retVal = 0
+    curl_out = "str"
+    tempf = tempfile.TemporaryFile()
+
+    print(cromwell_config.server)
+    # Check if Cromwell Server Backend works
+    HTTPUtils.assertCanCommunicateWithServer(cromwell_config.server)
+
+    # Curl workflow status
+    try:
+        curl_out = requests.get("{}/api/workflows/v1/{}/status".format(cromwell_config.server, workflow_id))
+    except requests.exceptions.RequestException as e:
+        print("Error: Cannot communicate with Cromwell server: {}".format(cromwell_config.server), e)
+        quit()
+    tempf.write(curl_out.content)
+
+    # tempf.seek(0) # test line delete post draft
+    # print(tempf.read())  # test line delete post draft
+
+    # Hold our status string here:
+    tempf.seek(0)
+    workflow_status_dic = json.loads(tempf.read())
```

Comment:
I wonder if you can directly ask `json` to load from `curl_out`, i.e. avoid writing to a temp file all together